elastic/kibana
# [Console] Output panel enhancements

## Summary

This PR enhances the output panel with a few QOL improvements such as:
* Adding a clear output button
* Request badges were moved to the bottom of the editor
* For multiple requests, the responses will also include the line number of the request
* Each response block will have a copy to clipboard button

##### How to test

* Start up es and kibana
* Navigate to `Devtools`
* Verify that:
  * Executing a single command shows the response output and that it can be copied to clipboard
  * Executing multiple requests, we should be able to select individual response outputs to copy their responses to the clipboard or also multiple response outputs
  * Clicking the `Clear this output` button should clear the devtools output


https://github.com/user-attachments/assets/93149cbc-181d-4dc6-9b7e-9c4acc854fe3

<details>
<summary>UI screenshots</summary>

![Screenshot 2024-08-19 at 16 03 35](https://github.com/user-attachments/assets/32cce2ff-aac7-48d5-a4ed-1d771b2ed1b7)
![Screenshot 2024-08-19 at 16 03 24](https://github.com/user-attachments/assets/b8abef8e-39bf-4226-98e8-54c4f2ce7d2d)
![Screenshot 2024-08-19 at 16 03 12](https://github.com/user-attachments/assets/8713e314-f532-4854-82a7-438198faa7e2)


</details>
